lib/runner.js

```diff
@@ -720,6 +721,47 @@ Runner.prototype.uncaught = function(err) {
 };
 
 /**
+ * Cleans up the reference to the test's deferred function.
+ * @see cleanSuiteReferences for details.
+ * @param {Test} test
+ */
+function cleanTestReferences(test) {
+  delete test.fn;
+}
+
+/**
+ * Cleans up the references to all the deferred functions (before/after/beforeEach/afterEach)
+ * of a Suite. These must be deleted otherwise it may create a memory leak as those functions may
+ * reference variables from closures, thus those variables can never be garbage collected as long
+ * as the deferred functions exist.
```

Comment:
The file appears to mostly follow a 80 char ruler - could you update this doc comment?